draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -216,8 +216,8 @@ querying via recursive resolvers.
 
 ## Properties
 
-Apart from catalog zone metadata stored at the apex (NS, SOA and the like), catalog zone information is stored in the form of "properties".
-Catalog consumers SHOULD ignore properties they do not understand.
+Catalog zone information is stored in the form of "properties".
+Catalog consumers SHOULD ignore records they do not understand.
```

Comment:
I like the change of the first line in the commit, but not the second. 
We already stated that "Catalog consumers SHOULD ignore any RR in the catalog zone which is meaningless or useless to the implementation." in the Description section. So this line is just a confirmation that this thus also holds for properties.

```suggestion
Catalog consumers SHOULD ignore properties they do not understand.
```

@mind04 WDYT?